deploy-edpm-reuse.yaml

```diff
@@ -2,6 +2,22 @@
 - name: Manage unique ID
   ansible.builtin.import_playbook: playbooks/unique-id.yml
 
+- name: Reboot controller-0 to make sure there are no running deployments
+  hosts: controller-0
+  gather_facts: false
+  tasks:
+    - name: Reboot controller-0
+      ansible.builtin.reboot:
+        reboot_timeout: 600
+      register: reboot_result
```

Comment:
Maybe you could drop this line and replace it instead with [post_reboot_delay](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/reboot_module.html#parameter-post_reboot_delay) to allow for the delay mentioned [here](https://github.com/openstack-k8s-operators/ci-framework/pull/2979/files#diff-fd2cd96602531f6ce6919f24b71046575a74b6dfeddaabcc00ee246d7393fa2cR17).
This would allow to perform the reboot, wait for it and continue with the `Wait for controller`.

Reply:
great idea, done